src/app/inventory/item-move-service.ts

```diff
@@ -838,7 +838,7 @@ function ItemService(): ItemServiceType {
           t(`ItemService.BucketFull.${moveAsideTarget.isVault ? 'Vault' : 'Guardian'}`, {
             itemtype,
             store: moveAsideTarget.name,
-            context: moveAsideTarget.gender && moveAsideTarget.gender.toLowerCase()
+            context: moveAsideTarget.gender?.toLowerCase()
```

Comment:
See #4732 for `i18next-scanner` requirements

Reply:
or even a little rewrite ala
```
const data =  {
            itemtype,
            store: moveAsideTarget.name,
            context: moveAsideTarget.gender?.toLowerCase()
          }
        const error: DimError = new Error(
          // t('ItemService.BucketFull.Guardian_male')
          // t('ItemService.BucketFull.Guardian_female')
          moveAsideTarget.isVault ? t(`ItemService.BucketFull.Vault', data) : t(`ItemService.BucketFull.Guardian', data);
        );
```

Reply:
as long as no special variables to i18next-scanner (count, etc) which inform it to add `_plural` this can most likely be used.

Reply:
this is implemented but i don't know how to test these messages without fully filling my chars and inventory to 500